README.md

```diff
@@ -74,6 +74,44 @@ Then update the `package.json` file in your project with the new version:
   }
 ```
 
+### Working with the Pattern Library
+
+Employer Style Base includes a [pattern library](http://cb-talent-development.github.io/employer-style-base/) built with [Fractal](http://fractal.build/). The pattern library serves as a quick reference to the components that come with style base and how to use them.
+
+If you haven't already, from within the employer-style-base repo run
+
+```
+yarn install
+```
+
+This will install Fractal and Fractal CLI.
+
+In order to show styled components, you must first compile from `.scss` to `.css`. Run
+
+```
+npm dev
```

Comment:
I think you'll want this to say `yarn run dev` - that's what I had to use to get this running locally

Edit: you may want to dileneate between when you'd use `yarn run sass:build` vs when you'd use `yarn run sass:watch` here